content/en/real_user_monitoring/guide/send-rum-custom-actions.md

```diff
@@ -4,45 +4,90 @@ kind: guide
 further_reading:
 - link: '/real_user_monitoring/explorer'
   tag: 'Documentation'
-  text: 'Visualize your RUM data in the Explorer'
+  text: 'Visualize your RUM data in the RUM Explorer'
 aliases:
   - /real_user_monitoring/guide/send-custom-user-actions/
 ---
 ## Overview
 
-Real User Monitoring [automatically collects actions][1] on your web application. You may also want to collect additional events and timings such as form completions and business transactions. With custom RUM actions, monitor any interesting event with all the relevant context attached. As an example throughout this guide, user checkouts information from an e-commerce website is collected.
+Real User Monitoring [automatically collects actions][1] on your web application. You can collect additional events and timings such as form completions and business transactions. 
+
+Custom RUM actions allow you to monitor interesting events with all the relevant context attached. For example, collecting a user's checkout information on an e-commerce website.
 
 ## Instrument your code
-To create a new RUM action, use the `addAction` API. Give your action a name and then attach context attributes in the form of a JavaScript object. In the following example, a `checkout` action is created with details about the user cart when the user clicks on the checkout button.
+
+Create a RUM action using the `addAction` API. Give your action a name and attach context attributes in the form of a JavaScript object. 
+
+The following example creates a `checkout` action with details about the user cart when the user clicks on the checkout button.
+
+{{< tabs >}}
+{{% tab "NPM" %}}
+
+```javascript
+import { datadogRum } from '@datadog/browser-rum';
+
+function onCheckoutButtonClick(cart) {
+    datadogRum.addAction('checkout', {
+        'value': cart.value, // for example, 42.12
+        'items': cart.items, // for example, ['tomato', 'strawberries']
+    })
+}
+```
+
+{{% /tab %}}
+{{% tab "CDN async" %}}
+```javascript
+DD_RUM.onReady(function() {
+    DD_RUM.addAction('<NAME>', '<JSON_OBJECT>');
+})
+
+function onCheckoutButtonClick(cart) {
+    DD_RUM.addAction('checkout', {
+        'value': cart.value, // for example, 42.12
+        'items': cart.items, // for example, ['tomato', 'strawberries']
+    })
+}
+```
+{{% /tab %}}
+{{% tab "CDN sync" %}}
 
 ```javascript
+window.DD_RUM && DD_RUM.addAction('<NAME>', '<JSON_OBJECT>');
+
 function onCheckoutButtonClick(cart) {
     DD_RUM.addAction('checkout', {
         'value': cart.value, // for example, 42.12
-        'items': cart.items, // efor example, ['tomato', 'strawberries']
+        'items': cart.items, // for example, ['tomato', 'strawberries']
     })
 }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
- example only to be consistent with npm tab
- ensure to check for DD_RUM existence before api call

```javascript
function onCheckoutButtonClick(cart) {
    window.DD_RUM && DD_RUM.addAction('checkout', {
        'value': cart.value, // for example, 42.12
        'items': cart.items, // for example, ['tomato', 'strawberries']
    })
}
```